sidebarsTheolive.ts

```diff
@@ -3,48 +3,27 @@ import type { SidebarsConfig } from '@docusaurus/plugin-content-docs';
 const sidebars: SidebarsConfig = {
   theolive: [
     'index',
+    'introduction-backend',
+    'introduction-player',
```

Comment:
Perhaps make an "introduction" category or something?

Or you can split the whole sidebar into backend and player, like this:
```typescript
const sidebars: SidebarsConfig = {
  theolive: [
    'index',
    {
      type: 'category',
      label: 'Player',
      collapsible: false,
      items: ['introduction-player'],
    },
    {
      type: 'category',
      label: 'Backend',
      collapsible: false,
      items: [
        'introduction-backend',
        // ...everything else
      ],
    },
  ],
};
```